integrations/jenkins/.port/resources/port-app-config.yaml

```diff
@@ -0,0 +1,43 @@
+createMissingRelatedEntities: true
+deleteDependentEntities: true
+resources:
+  - kind: job
+    selector:
+      query: "true"
+    port:
+      entity:
+        mappings:
+          identifier: .fullName | gsub(" "; "") | gsub("/"; "-") | ascii_downcase
+          title: .fullName
+          blueprint: '"jenkinsJob"'
+          properties:
+            jobName: .name
+            url: .url
+            jobStatus: '{"notbuilt": "created", "blue": "passing", "red": "failing"}[.color // "updated"]'
```

Comment:
updated is not a key in this jq

Reply:
I put it there in case a new status ever appears.

Reply:
no need pls remove it

also you are not checking if it exists and only then setting to updated
ur setting to updated if there is no color
this does not make sense

Reply:
removed